features/schema_metadata.feature

```diff
@@ -0,0 +1,296 @@
+      echo "Comment: " . $table->comment() . "\n";
+      echo "Compaction Class: " . $table->compactionStrategyClassName() . "\n";
+      foreach ($table->compactionStrategyOptions() as $key => $value) {
+        echo "  " . $key . ":" . $value . "\n";
+      }
+      foreach ($table->compressionParameters() as $key => $value) {
+        echo "  " . $key . ":" . $value . "\n";
+      }
+      echo "DC/Local Read Repair Chance: " . $table->localReadRepairChance() . "\n";
+      echo "Garbage Collection Grace Seconds: " . $table->gcGraceSeconds() . "\n";
+      echo "Read Repair Chance: " . $table->readRepairChance() . "\n";
+      """
+    When it is executed
+    Then its output should contain:
+      """
+      Name: values
+      Bloom Filter: 0.5
+      Caching: ALL
+      Comment: Schema Metadata Feature
+      Compaction Class: org.apache.cassandra.db.compaction.LeveledCompactionStrategy
+        sstable_size_in_mb:37
+        sstable_compression:org.apache.cassandra.io.compress.DeflateCompressor
+      DC/Local Read Repair Chance: 0.25
+      Garbage Collection Grace Seconds: 3600
+      Read Repair Chance: 0.75
+      """
+
+    @cassandra-version-less-2.1
+    Scenario: Additional table metadata will only exist for Cassandra versions
+              1.2 and v2.0
+    Given the following example:
+      """php
+      <?php
+      $cluster = Cassandra::cluster()
+                         ->withContactPoints('127.0.0.1')
+                         ->build();
+      $session = $cluster->connect("simplex");
+      
+      $schema  = $session->schema();
+      $table   = $schema->keyspace("simplex")->table("values");
+
+      echo "Populate I/O Cache on Flush: " . ($table->populateIOCacheOnFlush() ? "True" : "False") . "\n";
+      echo "Replicate on Write: " . ($table->replicateOnWrite() ? "True" : "False") . "\n";
+      """
+    When it is executed
+    Then its output should contain:
+      """
+      Populate I/O Cache on Flush: True
+      Replicate on Write: False
+      """
+
+    @cassandra-version-only-2.0
+    Scenario: Additional table metadata will only exist for Cassandra versions
+              2.0
+    Given the following example:
+      """php
+      <?php
+      $cluster = Cassandra::cluster()
+                         ->withContactPoints('127.0.0.1')
+                         ->build();
+      $session = $cluster->connect("simplex");
+      $cql = "ALTER TABLE values WITH index_interval = '512'";
+      $session->execute(new Cassandra\SimpleStatement($cql));
+      $schema  = $session->schema();
+      $table   = $schema->keyspace("simplex")->table("values");
+
+      echo "Index Interval: " . $table->indexInterval() . "\n";
+      """
+    When it is executed
+    Then its output should contain:
+      """
+      Index Interval: 512
+      """
+
+    @cassandra-version-2.0
+    Scenario: Additional table metadata will only exist for Cassandra versions
```

Comment:
Scenario name suggestion - Getting table metadata for default TTL, memtable flush period and speculative retry